lib/integer.gi

```diff
@@ -1195,7 +1195,11 @@ InstallGlobalFunction(PowerModInt,function ( r, e, m )
     # return the power
     return pow;
 end);
-
+if IsBound(POWERMODINT) then
```

Comment:
Is there a good reason to keep POWERMODINT_GAP around? Is POWERMODINT ever not going to be defined?